superset/daos/chart.py

```diff
@@ -39,6 +39,20 @@
 class ChartDAO(BaseDAO[Slice]):
     base_filter = ChartFilter
 
+    @classmethod
+    def get_filterable_columns_and_operators(cls) -> Dict[str, List[str]]:
+        filterable = super().get_filterable_columns_and_operators()
+        # Add custom fields for charts
+        filterable.update(
+            {
```

Comment:
NIT: also could fit as a module constant, this way it's not regenerated at every call